Tasks/PublishTestResults/publishtestresults.ts

```diff
@@ -1,56 +1,90 @@
+import * as fs from 'fs';
+import * as os from 'os';
 import * as path from 'path';
+import * as publishExe from './publishresultsthroughexe';
 import * as tl from 'vsts-task-lib/task';
+import * as tr from 'vsts-task-lib/toolrunner';
+import * as vsts from 'vso-node-api';
 import { publishEvent } from './cieventlogger';
 
 const MERGE_THRESHOLD = 100;
```

Comment:
can we have build variable to override explicitly?

Reply:
Added PublishTestResults.OverrideExeFlow which if is set to true won't hit the exe flow.